lib/srv/db/server.go

```diff
@@ -541,16 +568,39 @@ func (s *Server) startDatabase(ctx context.Context, database types.Database) err
 }
 
 // stopDatabase uninitializes the database with the specified name.
-func (s *Server) stopDatabase(ctx context.Context, name string) error {
+func (s *Server) stopDatabase(ctx context.Context, db types.Database) error {
 	// Stop database object importer.
-	if err := s.cfg.DatabaseObjects.StopImporter(name); err != nil {
-		s.log.WarnContext(ctx, "Failed to stop database object importer.", "db", name, "error", err)
+	if err := s.cfg.DatabaseObjects.StopImporter(db.GetName()); err != nil {
+		s.log.WarnContext(ctx, "Failed to stop database object importer",
+			"db", log.StringerAttr(db),
+			"error", err,
+		)
 	}
-	s.stopDynamicLabels(name)
-	if err := s.stopHeartbeat(name); err != nil {
-		return trace.Wrap(err)
+	s.stopDynamicLabels(db.GetName())
+
+	var errors []error
+	if err := s.stopHeartbeat(db.GetName()); err != nil {
+		s.log.WarnContext(ctx, "Failed to stop database heartbeat",
+			"db", log.StringerAttr(db),
+			"error", err,
+		)
+		errors = append(errors, err)
+	}
+
+	if err := s.stopHealthCheck(db); err != nil {
+		s.log.WarnContext(ctx, "Failed to stop database health checker",
+			"db", log.StringerAttr(db),
+			"error", err,
+		)
+		errors = append(errors, err)
 	}
-	s.log.DebugContext(ctx, "Stopped database.", "db", name)
+
+	if len(errors) > 0 {
+		return trace.NewAggregate(errors...)
```

Comment:
just curious if the caller is logging the aggregated errors or not. and if so, what level it is printing